test/cypress/integration/consumer-tools/obtener-respuestas/obtener-respuestas-helpers.js

```diff
@@ -0,0 +1,26 @@
+export class ObtenerRespuestasBuscar {
+
+  open() {
```

Comment:
We could consider ditching this `open` helper completely and just calling `cy.visit(…)` where it's needed. Being one line, the helper doesn't seem to be adding much. It would only make sense for test suites that open the same URL multiple times in several places (e.g. outside of a `beforeEach`).

Reply:
Just a thought. Doesn't need to be removed here.